pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1161,27 +1161,31 @@ private void verifyIndexingConfig(String tableNameWithType, IndexingConfig index
     }
   }
 
-  private void ensureRealtimeClusterIsSetUp(TableConfig config, String realtimeTableName,
-      IndexingConfig indexingConfig) {
-    StreamConfig streamConfig = new StreamConfig(realtimeTableName, indexingConfig.getStreamConfigs());
-    IdealState idealState = _helixAdmin.getResourceIdealState(_helixClusterName, realtimeTableName);
+  private void ensureRealtimeClusterIsSetUp(TableConfig realtimeTableConfig) {
+    String realtimeTableName = realtimeTableConfig.getTableName();
+    IdealState idealState = getTableIdealState(realtimeTableName);
+    // Skip processing disabled IdealState
```

Comment:
We should be able to add/remove llc even if table is disabled, I would think

Reply:
Correct. Removed the enable check